UIDD/UIDD-Session4/Assignment-1/css/styles.css

```diff
@@ -0,0 +1,86 @@
+.container{
+	margin: auto;
+	width: 80%;
+	background-color: #e6e6e6;
+	padding: 10px;
+	box-sizing: border-box;
+	border-radius: 10px;
+	overflow: hidden;
+}
+
+.row{
+	width: 100%;
+	padding: 5px;
+	display: flex;
+	justify-content: center;
+}
+
+input, select, textarea , button{
+	width: 40%;
+	padding: 10px;
+	box-sizing: border-box;
+	border-radius: 3px;
+	font-family: inherit;
+	font-size: 100%;
+	border: 1px solid #bab3b3;
+	background-color: whitesmoke;
+	font-weight: bold;
+}
+
+button{
+	width: auto;
+}
+
+input[type=radio]:valid {
+	width: auto;
+}
+
+.btn{
+	color: white;
+	background-color: orange;
+}
+
+.label{
+	width: 40%;
+	text-align: right;
+	padding-right: 6%;
+	box-sizing: border-box;
+	font-weight: bold;
+}
+
+.content{
+	width: 50%;
+}
+
+.icon{
+	border: 1px solid #bab3b3;
+	padding: 10px 15px;
+	box-sizing: border-box;
+	border-radius: 3px;
+	color: gray;
+}
+
+#btn{
+	width: 166px;
+}
+
+#project_icon{
+	padding: 19px 15px;
+}
+
+input[type=text]:valid {
```

Comment:
input[type=text]:valid {
	border-color: green;
}
 input[type=email]:valid {
	border-color: green;
}
These two can be combined as having same style.
input[type=text]:valid, input[type=email]:valid {
	border-color: green;
}